anubis-management-api/src/wac.py

```diff
@@ -12,6 +12,23 @@
 # user urls should be /{realm}/users/{id}
 # group urls should be /{realm}/groups/{id}
 
+def parse_rdf_graph(data):
+    g = Graph()
+    g.parse(data=data)
+    policies = {}
+    for subj, pred, obj in g:
+        policy = str(subj).split("/")[-1]
+        if not policies.get(policy):
+            policies[policy] = {}
+        if "agentClass" in str(pred):
+            policies[policy]["agentClass"] = str(obj)
+        if "mode" in str(pred):
+            policies[policy]["mode"] = str(obj)
+        if "acl#default" in str(pred):
+            policies[policy]["accessTo"] = "default"
```

Comment:
the parsing does not seem generic.
- what about `accessTo` policies?
- what about supporting the uri configuration by stripping the base url outs before storing in the db?